packages/mermaid/src/diagrams/mindmap/mindmapRenderer.ts

```diff
@@ -155,38 +158,33 @@ function positionNodes(cy) {
   });
 }
 
-/**
- * Draws a an info picture in the tag with id: id based on the graph definition in text.
- *
- * @param {any} text
- * @param {any} id
- * @param {any} version
- * @param diagObj
- */
-
-export const draw = async (text, id, version, diagObj) => {
+export const draw = async (text: string, id: string, version: string, diagObj: Diagram) => {
   const conf = getConfig();
-
+  const db = diagObj.db as MindmapDB;
   conf.htmlLabels = false;
 
   log.debug('Rendering mindmap diagram\n' + text, diagObj.parser);
 
-  const securityLevel = getConfig().securityLevel;
+  const securityLevel = conf.securityLevel;
   // Handle root and Document for when rendering in sandbox mode
-  let sandboxElement;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let sandboxElement: any;
   if (securityLevel === 'sandbox') {
     sandboxElement = select('#i' + id);
   }
-  const root =
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const root: any =
     securityLevel === 'sandbox'
       ? select(sandboxElement.nodes()[0].contentDocument.body)
       : select('body');
-  // Parse the graph definition
 
   const svg = root.select('#' + id);
```

Comment:
We already have a [function](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/rendering-util/selectSvgElement.ts) that do all of this:

```typescript
import { selectSvgElement } from '../../rendering-util/selectSvgElement.js';

const svg: SVG = selectSvgElement(id);
```